include/inviwo/core/io/serialization/serialization.h

```diff
@@ -27,13 +27,8 @@
  *
  *********************************************************************************/
 
-#ifndef IVW_SERIALIZATION_H
-#define IVW_SERIALIZATION_H
+#pragma once
 
-#include <inviwo/core/io/serialization/serializebase.h>
-#include <inviwo/core/io/serialization/serializer.h>
 #include <inviwo/core/io/serialization/serializable.h>
+#include <inviwo/core/io/serialization/serializer.h>
```

Comment:
is there any reason to keep this header?

Reply:
to avoid breaking stuff?